client/src/routes/index.svelte

```diff
@@ -18,7 +58,7 @@
 <section class="main">
 	<div class="wrapper">
 		<h1>WebMP: Mentor growth</h1>
-		<p>Some small description to make people feel inspired and attracted towards the service.</p>
+		<p>Some small description to make people feel inspired and attracted towards the service. Lorem ipsum dolor sit amet consectetur adipisicing elit. Harum qui itaque omnis ut ex alias ducimus.</p>
```

Comment:
Let's remove the Latin and put some text of our own here

Reply:
Sure Nivedit, great suggestion!